README.md

```diff
@@ -158,6 +162,8 @@ inspired by SwiftUI. For example, if you have a feature model like the one discu
 the [SwiftUI](#swiftui) section:
 
 ```swift
+import SwiftUINavigation 
+
```

Comment:
Should this be `SwiftNavigation`? And wanna add the `@CasePathable` to the `Destination` enum?

```suggestion
import SwiftNavigation 

```

Reply:
You are right. Minimum required is SwiftNavigation